Index: PhysicsTools/TagAndProbe/interface/BaseTreeFiller.h
===================================================================
RCS file: /local/reps/CMSSW/CMSSW/PhysicsTools/TagAndProbe/interface/BaseTreeFiller.h,v
retrieving revision 1.12
diff -u -8 -p -r1.12 BaseTreeFiller.h
--- PhysicsTools/TagAndProbe/interface/BaseTreeFiller.h	13 Dec 2011 02:32:34 -0000	1.12
+++ PhysicsTools/TagAndProbe/interface/BaseTreeFiller.h	21 Aug 2012 10:22:19 -0000
@@ -172,17 +172,17 @@ class BaseTreeFiller : boost::noncopyabl
 	bool addEventVariablesInfo_;
 
         void addBranches_(TTree *tree, const edm::ParameterSet &iConfig, const std::string &branchNamePrefix="") ;
 
         //implementation notice: these two are 'mutable' because we will fill them from a 'const' method
         mutable TTree * tree_;
         mutable float weight_;
         mutable float PUweight_;
-        mutable uint32_t run_, lumi_, event_, mNPV_;
+        mutable uint32_t run_, lumi_, bx_, event_, mNPV_;
 
         mutable float mPVx_,mPVy_,mPVz_,mBSx_,mBSy_,mBSz_; 
 
         mutable float mMET_,mSumET_,mMETSign_,mtcMET_,mtcSumET_,
 	  mtcMETSign_,mpfMET_,mpfSumET_,mpfMETSign_;
 };
 
 
