Index: MuonAnalysis/TagAndProbe/python/common_variables_cff.py
===================================================================
RCS file: /local/reps/CMSSW/CMSSW/MuonAnalysis/TagAndProbe/python/common_variables_cff.py,v
retrieving revision 1.35
diff -u -8 -p -r1.35 common_variables_cff.py
--- MuonAnalysis/TagAndProbe/python/common_variables_cff.py	4 Jul 2012 17:12:57 -0000	1.35
+++ MuonAnalysis/TagAndProbe/python/common_variables_cff.py	21 Aug 2012 10:26:12 -0000
@@ -105,16 +105,17 @@ TriggerVariables = cms.PSet(L1Variables,
 AllVariables = cms.PSet(KinematicVariables, IsolationVariables, MuonIDVariables, MuonCaloVariables, TrackQualityVariables, GlobalTrackQualityVariables, L1Variables, L2Variables, L3Variables)
 
 TrackQualityFlags = cms.PSet(
     Track_HP  = cms.string("? track.isNonnull ? track.quality('highPurity') : 0"),
 )
 MuonIDFlags = cms.PSet(
     Calo   = cms.string("isCaloMuon"),
     Glb    = cms.string("isGlobalMuon"),
+    GlbOrTMwMatch = cms.string('(isGlobalMuon || (isTrackerMuon && numberOfMatches>0))'),
     GlbPT  = cms.string("muonID('GlobalMuonPromptTight')"),
     TM     = cms.string("isTrackerMuon"),
     TMA    = cms.string("muonID('TrackerMuonArbitrated')"),
     PF     = cms.string("isPFMuon()"),
     TMLSAT = cms.string("muonID('TMLastStationAngTight')"),
     TMLST  = cms.string("muonID('TMLastStationTight')"),
     TMOSL  = cms.string("muonID('TMOneStationLoose')"),
     TMOST  = cms.string("muonID('TMOneStationTight')"),
